Makefile.inc

```diff
@@ -54,7 +54,7 @@ ifndef SYSTEMD
 endif
 
 ifndef SYSTEMDPATH
-	SYSTEMDPATH=usr/lib
+	SYSTEMDPATH=lib
```

Comment:
no, this is incorrect. We should rather set SYSTEMDPATH where we set the other paths.

Reply:
Sorry, this wasn't "incorrect". But still I'd like to do this differently.

For the time being, you can just build with `make SYSTEMDPATH=/lib`, no?